using-mobile-extensions/adobe-media-analytics/README.md

```diff
@@ -0,0 +1,155 @@
+# Adobe Media Analytics for Audio and Video
+
+## **Configure Media Analytics Extension in Launch**
+
+1. In Launch, click the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Media Analytics for Audio and Video** extension and click **Install**.
+3. Provide extension settings \(see [Configure Media Analytics Extension](#configure-media-analytics-extension)\).
+4. Click **Save**.
+5. Follow the publishing process to update your SDK configuration.
+
+### **Configure Media Analytics Extension**
+
+![Adobe Media Analytics Extension Configuration](../../.gitbook/assets/ext-ma-configuration.png)
+
+#### **Tracking Server**
+
+{% hint style="info" %}
+This is different from your Analytics tracking server.
+{% endhint %}
+
+Provide the tracking server to which all media tracking data should be sent.
+
+#### **Channel**
+
+Channel name property.
+
+#### **Online Video Provider**
+
+Name of the online platform through which content gets distributed.
+
+#### **Player Name**
+
+Name of the media player in use (e.g., "AVPlayer", "Native Player", "Custom Player").
+
+#### **Application Version**
+
+The version of the media player application/SDK.
+
+#### **Debug Logging**
+
+Enables or disables Media SDK logs.
+
+{% hint style="danger" %}
+This should be disabled for your production application.
+{% endhint %}
+
+### Add Media Analytics to your app
+
+{% hint style="info" %}
+This extension requires the [Adobe Analytics Extension](../adobe-analytics/README.md). You must add the Analytics extension to your Launch property, and make sure the extension is configured correctly.
```

Comment:
- Please remove the comma between "property" and "and".
- Please move "correctly" to before "configured".